docs/graphics/data-viz-color-examples/dataVizSuccessError.svg

```diff
@@ -1 +1 @@
683-1.101 1.558-1.101.889 0 1.463.39 1.62.978h1.162c-.157-1.21-1.23-1.976-2.775-1.976-1.565 0-2.748.923-2.748 2.146Z"/><path fill="var(--color-text-icon-subtle)" fill-rule="evenodd" d="M474 78a1 1 0 1 1-.001-1.999A1 1 0 0 1 474 78Zm1 5a1 1 0 1 1-2 0v-3.333a1 1 0 1 1 2 0V83Zm-1-11a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z" clip-rule="evenodd"/><path fill="var(--color-text-default)" d="M424.68 115.812c3.144 0 5.429-2.08 5.429-5 0-2.646-1.865-4.589-4.589-4.589-1.905 0-3.291.957-3.838 2.265h-.059c-.01-3.086 1.006-5.068 3.057-5.068 1.162 0 2.002.586 2.334 1.592h2.91c-.391-2.324-2.5-3.926-5.235-3.926-3.662 0-5.888 2.812-5.888 7.48 0 4.864 2.51 7.246 5.879 7.246Zm-.039-2.343c-1.446 0-2.579-1.133-2.579-2.588 0-1.436 1.114-2.481 2.608-2.481 1.475 0 2.568 1.055 2.568 2.539-.009 1.416-1.152 2.53-2.597 2.53Zm10.634-4.17h1.612c1.709 0 2.715.762 2.705 2.129 0 1.191-1.016 2.011-2.49 2.011-1.553 0-2.569-.781-2.666-1.933h-2.813c.127 2.588 2.295 4.306 5.479 4.306 3.271 0 5.488-1.699 5.488-4.257 0-1.934-1.367-3.174-3.457-3.369v-.059c1.66-.342 2.939-1.504 2.939-3.301 0-2.256-2.011-3.73-4.99-3.73-2.949 0-5 1.65-5.146 4.228h2.705c.088-1.22 1.054-1.963 2.441-1.963 1.406 0 2.197.733 2.197 1.885 0 1.152-.957 1.953-2.431 1.953h-1.573v2.1Zm14.698 6.513c3.574 0 5.742-2.841 5.742-7.392s-2.188-7.334-5.742-7.334c-3.555 0-5.752 2.793-5.752 7.344 0 4.56 2.177 7.382 5.752 7.382Zm0-2.363c-1.739 0-2.754-1.787-2.754-5.019 0-3.203 1.035-4.981 2.754-4.981 1.728 0 2.744 1.768 2.744 4.981 0 3.232-1.006 5.019-2.744 5.019Z"/><path fill="var(--color-data-visualization-error-text)" fill-rule="evenodd" d="M476.057 110.39a1.332 1.332 0 1 1 1.885-1.885l1.724 1.724V102a1.333 1.333 0 0 1 2.667 0v8.229l1.724-1.724a1.332 1.332 0 1 1 1.885 1.885l-4.943 4.944-4.942-4.944Z" clip-rule="evenodd"/><path fill="var(--color-data-visualization-error-text)" d="M493.882 106.13h1.914c0-.964.629-1.641 1.634-1.641.964 0 1.524.643 1.524 1.449 0 .684-.266 1.135-1.497 2.332l-3.459 3.342V113h7.116v-1.661h-4.334v-.123l2.051-1.955c1.641-1.559 2.147-2.352 2.147-3.507 0-1.661-1.34-2.871-3.432-2.871-2.16 0-3.664 1.326-3.664 3.247Zm10.848 1.606c1.477 0 2.345-.882 2.345-2.399 0-1.483-.888-2.393-2.345-2.393-1.449 0-2.344.916-2.344 2.393 0 1.511.868 2.399 2.344 2.399Zm1.012 5.264 6.556-9.864h-1.579L504.163 113h1.579Zm-1.012-6.453c-.499 0-.786-.438-.786-1.203 0-.745.294-1.203.786-1.203.486 0 .78.458.78 1.203 0 .759-.281 1.203-.78 1.203Zm7 6.562c1.477 0 2.345-.881 2.345-2.399 0-1.483-.888-2.393-2.345-2.393-1.449 0-2.344.916-2.344 2.393 0 1.511.875 2.399 2.344 2.399Zm0-1.189c-.499 0-.779-.438-.779-1.203 0-.745.287-1.203.779-1.203.493 0 .787.458.787 1.203 0 .759-.288 1.203-.787 1.203Z"/><path stroke="var(--color-data-visualization-error-graph)" stroke-linecap="round" stroke-linejoin="round" stroke-width="3" d="m418 177.203 9.435-4.603 8.845 4.603 10.467-11.762 8.845 17.388 9.73-48.329 10.319 14.575 8.993 41.425 10.024-7.671h8.551l9.582-30.174 10.319 12.786h8.551l9.287 14.319L558 173.93"/><path stroke="#E9E9E9" d="M.5.5h735v261H.5z"/></svg>
```

Comment:
@ayeshakmaz I dont see anything here?

Reply:
It's changes to some of the tokens used in the code (text color mainly)